DFC.App.JobProfileTasks.SegmentService/JobProfileTasksSegmentService.cs

```diff
@@ -219,6 +225,8 @@ public async Task<UpsertJobProfileTasksModelResponse> UpsertAsync(JobProfileTask
 
             var result = await repository.UpsertAsync(tasksSegmentModel).ConfigureAwait(false);
```

Comment:
thsi line is no longer required as the upsert is called in the UpsertAndRefreshSegmentModel method - please remove

Reply:
This has been removed